tests/library/inspector/cli-codegen-1.spec.ts

```diff
@@ -624,6 +624,93 @@ await page.Locator(\"#age\").SelectOptionAsync(new[] { \"2\" });`);
     expect(message.text()).toBe('2');
   });
 
+  const clickMultipleSelectOption = async (locator: Locator, withCtrl = false) => {
+    const page = locator.page();
+
+    // Webkit can't click multiple select options
+    // https://github.com/microsoft/playwright/issues/32126
+    if (page.context().browser().browserType().name() === 'webkit') {
+      const elem = await locator.elementHandle();
+      const rect = await elem!.evaluate(e => {
+        return e.getBoundingClientRect()!;
+      });
+      if (withCtrl)
+        await page.keyboard.down('Control');
+
+      await page.mouse.click(rect.x + rect.width / 2, rect.y + rect.height / 2);
+      if (withCtrl)
+        await page.keyboard.up('Control');
+
+    } else {
+      await locator.click({ modifiers: withCtrl ? ['Control'] : [] });
+    }
+  };
+
+  test('should select with multiple attribute', async ({ page, openRecorder }) => {
+    const recorder = await openRecorder();
+
+    await recorder.setContentAndWait(`<select id="age" multiple onchange="console.log('[' + [...age.selectedOptions].map(x => x.value).join(',') + ']')"><option value="1">1</option><option value="2">2</option></select>`);
+
+    const locator = await recorder.hoverOverElement('select');
+    expect(locator).toBe(`locator('#age')`);
+    await clickMultipleSelectOption(page.getByRole('option', { name: '1' }));
```

Comment:
I hope clicking the `select` instead of `option` with offset to pick an element would work across the browsers.

Reply:
@pavelfeldman 
Um, I'm not sure what kind of operation you're referring to. Could you show me a code example?